README.md

```diff
@@ -1,119 +1,84 @@
+  - Edit the config dev/config/ui/dev/build.yml, and modify the stanza for kbase-ui-plugin-data-landing-pages.
+    *Note* that this directory must be the parent of kbase-ui-plugin-data-landing-pages.
 ```
     -
         name: data-landing-pages
         globalName: kbase-ui-plugin-data-landing-pages
-        version: 0.1.0
+        version: x.y.z
         cwd: src/plugin
         source:
-            # uncomment the bower property to have the build fetch from bower/github
-            # bower: {}
-            # use something like this to feed bower a specific alternate version spec.
-            # in this case a specific github repo and branch
-            # bower:
-            #     version: you/kbase-ui-plugin-data-landing-pages#master
-            # comment this out if bower us activated
-            directory: 
-                # NB the root is the directory which contains this plugin, which
-                # must be located within a directory whose name is the globalName
-                root: /Users/you/work/kbase/sprints/data-api-widgets
-``` 
-    
-This setup will allow the "classic" edit-build-refresh workflow to integrate the plugin. Since the data api landing page repo contains just data widgets, there is no need for further configuration of the ui.
-
-If you hop back into kbase-ui and issue ```make build``` the plugin will be operational.
-
-You can test this by visiting a dataview url which should invoke an appropriate object (see the data api demo menu item which provides links to inspect compatible objects -- the references can pasted into the dataview url.)
-
-For example, one available in CI now is: http://localhost:8080/#dataview/1779/1006539/1
-
-### Faster Develop Workflow
-
-It is very simple to set up a faster edit-refresh workflow. Simply link your plugin directly into the build and refrain from running *make build*. For example, something like this from the parent directory of kbase-ui and the plugin:
+            #bower: {}
+            directory:
+                root: /absolute/path/to/parent/
+```
 
+4. Start kbase-ui
+This starts a static nodejs server at :8080.
 ```
-rm -rf kbase-ui/build/build/client/modules/plugins/data-landing-pages
-ln -s `pwd`/kbase-ui-plugin-data-landing-pages/src/plugin kbase-ui/build/build/client/modules/plugins/data-landing-pages
+make start
 ```
 
-## Registering in Bower
-
-[ coming soon ] 
-
-
-## Integrating into the Developer and Production Build
-
-### Developer Build
-
-The configuration file you copied into kbase-ui/dev will not be commited in to the project. This protects the project integrity, but also does not add your plugin to the ui either. To incorporate it into the ui, you just need to add a stanza as before, but this time have it loaded from bower:
-
-- edit kbase-ui/config/ui/dev.yml
-- add the following stanza to the plugins section:
+4a. (optional) Launch browser window
+This launches your default browser with a local instance of the kbase ui.
+```
+make preview
```

Comment:
This fails for symlinks:
"ERROR: /modules/plugins/data-landing-pages/config.yml:Not Found"